apps/test-suite/tests/Image.js

```diff
@@ -195,6 +195,19 @@ export async function test(t, { setPortalChild, cleanupPortal }) {
           t.expect(path).toBe(null);
         }
       });
+
+      t.it('prefetches an image with headers and resolves promise to true', async () => {
+        await Image.clearDiskCache();
+        const result = await Image.prefetch(REMOTE_KNOWN_SOURCE.uri, undefined, {
```

Comment:
We generally try to avoid using `undefined` explicitly. If you want to skip an argument, it's better to use `null` or the default value for this argument. Since both cache policy and headers are optionals, we can also consider merging them into a single options object 🤔 Like:

```ts
await Image.prefetch('url', {
  cachePolicy: 'disk',
  headers: { ... }
});
```

Reply:
Thanks for your review!
I think merging them into a single options object is a great idea.
I'll make them function overloads that are safe from breaking changes.